hubblestack/fileclient.py

```diff
@@ -492,17 +467,19 @@ def get_url(self, url, dest, makedirs=False, saltenv='base',
                 if makedirs:
                     os.makedirs(destdir)
                 else:
-                    return ''
+                    return ""
         elif not no_cache:
             dest = self._extrn_path(url, saltenv, cachedir=cachedir)
             if source_hash is not None:
                 try:
-                    source_hash = source_hash.split('=')[-1]
+                    source_hash = source_hash.split("=")[-1]
                     form = hubblestack.utils.files.HASHES_REVMAP[len(source_hash)]
                     if hubblestack.utils.hashutils.get_hash(dest, form) == source_hash:
                         log.debug(
-                            'Cached copy of %s (%s) matches source_hash %s, '
-                            'skipping download', url, dest, source_hash
+                            "Cached copy of %s (%s) matches source_hash %s, " "skipping download",
```

Comment:
Unnecessary " " in log.debug() between %s, and skipping
`"Cached copy of %s (%s) matches source_hash %s, skipping download",`